tests/functional/pv/pv_services/test_cephfs_pvc_with_chunk_size.py

```diff
@@ -0,0 +1,96 @@
+import time
+import pytest
+import logging
+from ocs_ci.framework.testlib import bugzilla, tier2
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import cluster
+from ocs_ci.framework.pytest_customization.marks import magenta_squad
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    argnames="interface",
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHFILESYSTEM], marks=pytest.mark.polarion_id("OCS-323")
+        ),
+    ],
+)
+@tier2
+@bugzilla("2171225")
+@magenta_squad
+class TestCephfsWithChunkIo:
+    """
+    This class takes care of create Cephfs PVC, create Fedora dc pod and run Chunk IO on fedora pod
+    """
+
+    def test_cephfs_with_large_chunk_io(self, interface, pvc_factory, dc_pod_factory):
+
+        """
+        This function facilitates
+        1. Create PVC with Cephfs, access mode RWX
+        2. Create dc pod with Fedora image
+        3. Copy helper_scripts/chunk.py to Fedora dc pod
+        4. Set debug 25 for mds in rook-ceph-tools pod
+        5. Run chunk.py on fedora pod for 1hr
+        6. Read /var/log/ceph/ceph-mds.****.log and look for errors
+        7. If no errors seen in the mds.log then the test will Pass.
+        8. If any errors found then the test will fail.
+        """
+        access_mode = constants.ACCESS_MODE_RWX
+        file = constants.CHUNK
+
+        # Creating PVC with cephfs as inetrface
+        log.info(f"Creating {interface} based PVC")
+        pvc_obj = pvc_factory(interface=interface, access_mode=access_mode, size="50")
+        # Creating a Fedora dc pod
+        log.info("Creating fedora dc pod")
+        pod_obj = dc_pod_factory(
+            pvc=pvc_obj, access_mode=access_mode, interface=interface
+        )
+        # Copy chunk.py to fedora pod
+        log.info("Copying chunk.py to fedora pod ")
+        cmd = f"oc cp {file} {pvc_obj.namespace}/{pod_obj.name}:/"
+        helpers.run_cmd(cmd=cmd)
+        log.info("chunk.py copied successfully ")
+
+        # set debug 25 in rook-ceph-tools for mds
+        ceph_cluster = cluster.CephCluster()
+        ceph_cluster.ceph_config_set_debug()
+        log.info("mds debug has been set to 25 successfully ")
+        # Run chunk.py on fedora pod
```

Comment:
I feel it will be a better idea to create an image and run the application pod using a deployment job. 
Deploy a pod using that image and execute the script inside the pod with the specified runtime.  Upon reaching the deployment job pod to completed state, we can conclude that the script executed successfully

Reply:
Image creation is not working for now, need to made changes and test. I feel that we can merge the code for now, this requirement can be a P2. Next time, I can create image and modify this test case. What do you think?